cgmanifest.json

```diff
@@ -666,8 +666,8 @@
         "type": "other",
         "other": {
           "name": "check",
-          "version": "0.12.0",
-          "downloadUrl": "https://github.com/libcheck/check/archive/0.12.0.tar.gz"
+          "version": "0.15.2",
+          "downloadUrl": "https://github.com/libcheck/check/archive/refs/tags/0.15.2.tar.gz"
```

Comment:
Why does this URL differ from the uncommented one from the .spec file? I'm a bit confused. If the actual (working) link is this one, then let's not provide invalid ones in the .spec file.